android/app/src/main/java/com/woowacourse/ody/presentation/date/MeetingDateFragment.kt

```diff
@@ -20,8 +23,39 @@ class MeetingDateFragment : Fragment() {
         return binding.root
     }
 
+    override fun onViewCreated(
+        view: View,
+        savedInstanceState: Bundle?,
+    ) {
+        super.onViewCreated(view, savedInstanceState)
+
+        initializeCalendar()
+    }
+
+    private fun initializeCalendar() {
+        val today = Calendar.getInstance().timeInMillis
+        binding.cvDate.minDate = today
+        binding.cvDate.setOnDateChangeListener { _, year, month, dayOfMonth ->
+            val selectedDate = Calendar.getInstance()
+            selectedDate.set(year, month, dayOfMonth)
+
+            if (selectedDate.timeInMillis < today) {
```

Comment:
저번에 해음이 얘기했던 거처럼, 사용자가 11:59에 이 화면에 진입해서 12:00이 되는 경우가 있을 수 있겠네요. `today` 객체는 리스너에서 계속해서 받아오는 건 어떨까요??
근데 이렇게 예외 상황을 생각하면 끝도 없을 것 같아서.. 이 리뷰는 반영하지 않아도 됩니다~!

```suggestion
        val today = Calendar.getInstance().timeInMillis
        binding.cvDate.minDate = today
        binding.cvDate.setOnDateChangeListener { _, year, month, dayOfMonth ->
            val selectedDate = Calendar.getInstance()
            selectedDate.set(year, month, dayOfMonth)

            if (selectedDate.timeInMillis < Calendar.getInstance().timeInMillis) {
```

Reply:
너무 좋습니다!